tests/rptest/tests/fix_5355_upgrade_test.py

```diff
@@ -45,7 +45,8 @@ def __init__(self, test_context):
     def setUp(self):
         # NOTE: `rpk redpanda admin brokers list` requires versions v22.1.x and
         # above.
-        self.installer.install(self.redpanda.nodes, (22, 1, 3))
+        self.oldversion, self.oldversion_str = self.installer.install(
+            self.redpanda.nodes, (22, 1))
```

Comment:
Given tests a bug fix that's been backported 22.1.x, it might be worth pinning this one to 22.1.3?